python/example_code/api-gateway/README.md

```diff
@@ -0,0 +1,116 @@
+# API Gateway code examples for the SDK for Python
+
+## Overview
+
+Shows how to use the AWS SDK for Python (Boto3) to work with Amazon API Gateway.
+
+<!--custom.overview.start-->
+<!--custom.overview.end-->
+
+*API Gateway enables you to create and deploy your own REST and WebSocket APIs at any scale.*
+
+## ⚠ Important
+
+* Running this code might result in charges to your AWS account.
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+<!--custom.important.start-->
+<!--custom.important.end-->
+
+## Code examples
+### Single actions
+
+Code excerpts that show you how to call individual service functions.
+
+* [Add a response from an AWS service integrated with a method](aws_service\aws_service.py#L25) (`PutIntegrationResponse`)
+* [Add an HTTP method to a REST resource](aws_service\aws_service.py#L25) (`PutMethod`)
+* [Add an HTTP response to a REST resource](aws_service\aws_service.py#L25) (`PutMethodResponse`)
+* [Create a REST API](aws_service\aws_service.py#L41) (`CreateRestApi`)
+* [Create a REST resource](aws_service\aws_service.py#L73) (`CreateResource`)
+* [Delete a REST API](aws_service\aws_service.py#L228) (`DeleteRestApi`)
+* [Deploy a REST API](aws_service\aws_service.py#L164) (`CreateDeployment`)
+* [Get REST resources](aws_service\aws_service.py#L25) (`GetResources`)
+* [Integrate a method with an AWS service](aws_service\aws_service.py#L94) (`PutIntegration`)
+* [List REST APIs](aws_service\aws_service.py#L201) (`GetRestApis`)
+
+### Scenarios
+
+Code examples that show you how to accomplish a specific task by calling multiple
+functions within the same service.
+
+* [Create and deploy a REST API](aws_service\aws_service.py) 
+
+### Cross-service examples
+
+Sample applications that work across multiple AWS services.
+
+* [Create a REST API to track COVID-19 data](../../cross_service/apigateway_covid-19_tracker) 
+* [Create a lending library REST API](../../cross_service/aurora_rest_lending_library) 
+* [Create a websocket chat application](../../cross_service/apigateway_websocket_chat) 
+
+## Run the examples
+
+### Prerequisites
+
+See the [README](../../README.md#Prerequisites) in the `python` folder for prerequisites.
```

Comment:
For prerequisites, see the [README](../../README.md#Prerequisites) in the `python` folder.

Reply:
fixed